web/src/pages/index.page.tsx

```diff
@@ -50,8 +54,30 @@ const Home: NextPage = () => {
 
       <section id="how-it-works">
         <StyledBox>
-          <Typography variant="h4">How it works</Typography>
-          <YoutubeEmbed embedId="yM8RrwQWeJc" />
+          <Box sx={{ height: 600, width: "70%" }}>
+            <Typography variant="h4">How it works</Typography>
+            <Carousel autoPlay={false}>
```

Comment:
It's a little jarring when the carousel changes size due to its contents changing. I think it'd be ideal to have its height be set to the largest child's height, then center the children within the carousel if they're smaller than the carousel, but I'm not sure how to accomplish this simply while allowing the carousel's size to also be responsive based on screen size.

I think it's fine to leave this issue unsolved for now.

Reply:
Yeah, I was annoyed by this behavior as well, but didn't find a good solution yet.

Reply:
maybe prioritize maintaining aspect ratio for now...